source/ios/tools/IOSFeed/IOSFeed.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+
+namespace IOSFeedNS
+{
+    class IOSFeed
+    {
+        public static void Main()
+        {
+            // Run the examples asynchronously, wait for the results before proceeding
+            ProcessAsync().GetAwaiter().GetResult();
+        }
+
+        private static async Task ProcessAsync()
+        {
+            string storageConnectionString;
+
+            var connectionStringPath = "source/ios/tools/IOSFeed/ConnectString.txt";
+            using (StreamReader sr = File.OpenText(connectionStringPath))
+            {
+                storageConnectionString = sr.ReadToEnd();
+            }
+
+            Console.WriteLine("key string: " + storageConnectionString);
+            // Check whether the connection string can be parsed.
+            CloudStorageAccount storageAccount;
+            if (CloudStorageAccount.TryParse(storageConnectionString, out storageAccount))
+            {
+                // If the connection string is valid, proceed with operations against Blob
+                // storage here.
+                // ADD OTHER OPERATIONS HERE
+                Console.WriteLine("success!");
+                // Create the CloudBlobClient that represents the 
+                // Blob storage endpoint for the storage account.
+                CloudBlobClient cloudBlobClient = storageAccount.CreateCloudBlobClient();
+
+                // Create a container called 'quickstartblobs' and 
+                // append a GUID value to it to make the name unique.
+
+                var containerId = "adaptivecardsiosblobs";
```

Comment:
>adaptivecardsiosblobs [](start = 35, length = 21)

This again can be a constant

Reply:
please look at review #4.

---
In reply to: [309841069](https://github.com/microsoft/AdaptiveCards/pull/3291#discussion_r309841069) [](ancestors = 309841069)